extensions/sql-database-projects/src/common/utils.ts

```diff
@@ -3,7 +3,40 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+import * as vscode from 'vscode';
 
 export function getErrorMessage(error: Error | string): string {
 	return (error instanceof Error) ? error.message : error;
 }
+
+export function trimUri(innerUri: vscode.Uri, outerUri: vscode.Uri): string {
+	let innerParts = innerUri.path.split('/');
```

Comment:
>/ [](start = 39, length = 1)

what about \ if it's linux path?

Reply:
Isn't Linux always `/`?  Windows is the one that uses `\` for file system paths and `/` for web paths.  Either way, I believe vscode.Uri resolves that concern for us by giving us access to both uri.path (what I'm using here, which should keep things consistent between platforms) and uri.fsPath (which follows the current platform's conventions).